javaagent-bootstrap/src/main/java/io/opentelemetry/javaagent/bootstrap/AgentInitializer.java

```diff
@@ -47,6 +47,7 @@
   // fields must be managed under class lock
   private static ClassLoader AGENT_CLASSLOADER = null;
 
+  // called via reflection in the OpenTelemetryAgent class
```

Comment:
I was working on understanding the startup process, and noticed that IDEA said this was unused, so it seemed worthwhile to add a comment. Can pull out of this PR if not appropriate.